code/__HELPERS/cmp.dm

```diff
@@ -136,3 +136,8 @@ GLOBAL_VAR_INIT(cmp_field, "name")
 
 /proc/cmp_mob_realname_dsc(mob/A,mob/B)
 	return sorttext(A.real_name,B.real_name)
+
+/// Orders bodyparts by their body_part value, ascending.
+/proc/cmp_bodypart_by_body_part_asc(obj/item/bodypart/limb_one, obj/item/bodypart/limb_two)
+	return limb_one.body_part - limb_two.body_part
```

Comment:
Could this be done with a define instead?
Seems very specific for a simple subtraction operation.

Reply:
yes/no, this gives more freedom for limbs and allows for things like quad arms to render properly if we ever do so. Basically fits with what klimbs was intended for.